doc/configuration.rst

```diff
@@ -140,6 +140,9 @@ If a module does not specify colors but it is in a container, then the colors
 of the container will be used if they are set, before using ones defined in the
 general section.
 
+Generally colors can specified using hex values eg ``#FF00FF`` or ``#F0F``.  It
+is also possible to use css3 color names eg ``red`` ``hotpink``.
+
```

Comment:
Can we have something like `For a valid list of css3 color names to use, see https://www.w3.org/TR/css-color-3/#svg-color` in there too?

Reply:
If we can find a nice list somewhere I'd really like one that includes `RebeccaPurple` which I think is actually css4 draft

Reply:
https://drafts.csswg.org/css-color/#named-colors :girl: :purple_heart: